rune/Makefile

```diff
@@ -161,8 +163,11 @@ localcross:
 	CGO_ENABLED=1 GOARCH=arm64 CC=aarch64-linux-gnu-gcc         $(GO_BUILD) -o runc-arm64 .
 	CGO_ENABLED=1 GOARCH=ppc64le CC=powerpc64le-linux-gnu-gcc   $(GO_BUILD) -o runc-ppc64le .
 
+rpm:
+	$(MAKE) -C dist/centos rpm RPMBUILD_DIR=$(RPMBUILD_DIR) INCLAVARE_CONTAINERS_VERSION=$(INCLAVARE_CONTAINERS_VERSION)
```

Comment:
Low level of Makefile will create RPMBUILD_DIR so it is unnecessary to pass it any more. Also, this layer doesn't use  RPMBUILD_DIR and why create it in this layer? The most top of RPMBUILD_DIR seems to be useless as well.